.gitignore

```diff
@@ -24,3 +24,5 @@ bin
 *~
 
 *sync.sh
+build.sh
+bundle
```

Comment:
We can add `bundle.Dockerfile` in here as well

```suggestion
bundle
bundle.Dockerfile
```

Reply:
Done.